urg_node/src/uam/uam_driver_ros.cpp

```diff
@@ -229,7 +294,8 @@ void UamROS::configureTimerCallback(const ros::TimerEvent& event)
   if (!configured_ && !configure())
   {
     // Configuration failed. Restart the timer to try again.
-    configure_timer_.start();
+    ROS_ERROR_STREAM("Failed to configure the lidar. Retrying in " << params_.reconfiguration_timeout.toSec()
```

Comment:
Why did you remove the timer? Timer is stopped at the top, this will never run again if it fails to connect to the lidar